<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>684</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>684</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DBA_to_DWD建表转换工具</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_dba">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>155</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DBA建表语句</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dwd">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>155</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DWD建表语句</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_transfer">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>255</y>
      <width>75</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>290</y>
      <width>75</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_savePath">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>52</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>保存路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_savePath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>136</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_author">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>470</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray</string>
    </property>
    <property name="text">
     <string>Author: zenanhe</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_dba">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>180</y>
      <width>270</width>
      <height>280</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_dwd">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>270</width>
      <height>280</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_sysName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>52</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>系统名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sysName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_databaseType">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>52</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>源库类型</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_mysql">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MySQL</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_oracle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Oracle</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_tbName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>52</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>解析表名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tbName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
